Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -14,6 +14,7 @@ enum ReaderModeState: String {
 enum ReaderModeTheme: String {
     case Light = "light"
     case Dark = "dark"
+    case Print = "print"
```

Comment:
What is this "Print" theme?

Also, we have "Auto" on Android, which switches between light and dark depending on the ambient light. I wonder if that's possible on iOS.